### PR TITLE
Create models

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: backend/models/Address.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import Location from './Location';
+
+export default class Address extends Model {
+    static get tableName() {
+        return 'Address';
+    }
+    static get jsonSchema() {
+        return {
+            type: 'object',
+            required: ['address_id'],
```
Comment (sentayhuJo) [inline backend/models/Address.js]: `address_id` should not be required as its already defined as Auto

File: backend/models/Branch.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import Address from './Address';
+
+export default class Branch extends Model {
+    static get tableName() {
+        return 'Branch';
+    }
+    static get jsonSchema() {
+        return {
+            type: 'object',
+            required: [' branch_id'],
```
Comment (sentayhuJo) [inline backend/models/Branch.js]: ditto

File: backend/models/Location.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import Address from './Address';
+
+export default class Location extends Model {
+    static get tableName() {
+        return 'Location';
+    }
+    static get jsonSchema() {
+        return {
+            type: 'object',
+            required: ['location_id'],
```
Comment (sentayhuJo) [inline backend/models/Location.js]: ditto

File: backend/models/Organisation.js [hunk 4]
```diff
@@ -11,12 +12,26 @@ export default class Organisation extends Model {
 
       properties: {
         org_id: { type: 'integer' },
-        ownerId: { type: ['integer', 'null'] },
+        branch_id: { type: ['integer', 'null'] },
```
Comment (sentayhuJo) [inline backend/models/Organisation.js]: Why `branch_id` here ?

File: backend/models/Service.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import Branch from './Branch';
+
+export default class Service extends Model {
+    static get tableName() {
+        return 'Service';
+    }
+    static get jsonSchema() {
+        return {
+            type: 'object',
+            required: ['service_id', 'service_name', 'process'],
```
Comment (sentayhuJo) [inline backend/models/Service.js]: `service_id` cant be required here

File: backend/package.json [hunk 6]
```diff
@@ -11,6 +11,7 @@
     "body-parser": "~1.18.2",
     "debug": "~2.6.9",
     "express": "~4.15.5",
+    "express-promise-router": "^3.0.1",
```
Comment (sentayhuJo) [inline backend/package.json]: Nice

File: backend/models/Organisation.js (hunk 4 above)
Comment (DwinaTech) [reply backend/models/Organisation.js]: removed this, that is right, it is misuse of it here.

File: backend/models/Service.js (hunk 5 above)
Comment (DwinaTech) [reply backend/models/Service.js]: Done

File: backend/models/Location.js (hunk 3 above)
Comment (DwinaTech) [reply backend/models/Location.js]: Done

File: backend/models/Branch.js (hunk 2 above)
Comment (DwinaTech) [reply backend/models/Branch.js]: Done

File: backend/models/Address.js (hunk 1 above)
Comment (DwinaTech) [reply backend/models/Address.js]: Done

File: backend/models/Address.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ export default class Address extends Model {
     static get jsonSchema() {
         return {
             type: 'object',
-            required: ['address_id'],
+          required: ['adress_line', 'city', 'postcode'],
```
Comment (kkarimi) [inline backend/models/Address.js]: `adress -> address`

**Pull request merged**